### PR TITLE
Changes the description for the survival box everyone gets from cryo-wakeup.

### PR DESCRIPTION
**Pull request by HaultyAnonie**
## About the Pull Request

crew survival kit > personal emergency kit with updated description.

## Why It's Good For The Game

Without doubt, the most popular box used **every** round by at least _one_ person. How has this not been caught yet?

## Changelog

:cl:
tweak: tweaks the description and name of the box everyone gets upon cryo-wakeup in their bag.
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
speed merge this asap @TheDarkElites

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -96,8 +96,8 @@
 	foldable = null //special form fitted boxes should not be foldable.
 
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "personal emergency kit"
+	desc = "A survival box containing a variety of supplies. This one contains oxygen, as shown by the red panel and two striped yellow lines."
```
Comment (TheDarkElites) [inline code/game/objects/items/weapons/storage/boxes.dm]:
```suggestion
	desc = "A survival box containing a variety of supplies. The red panel and two striped yellow lines indicate this one contains oxygen."
```

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (HaultyAnonie) [reply code/game/objects/items/weapons/storage/boxes.dm]: Wordy wording overly wording basis wording- ok.

**Comment by TheDarkElites** [comment]
> speed merge this asap @TheDarkElites

if you tell me to speed merge something im going to find you and turn your innards into a custard.

**Comment by HaultyAnonie** [comment]
> indicate this one contains oxygen

shouldn't this be 
> that this one contains

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (TheDarkElites) [reply code/game/objects/items/weapons/storage/boxes.dm]: Not resolved, your wording is overly wordy please write what I requested.

Comment (HaultyAnonie) [reply code/game/objects/items/weapons/storage/boxes.dm]: But- I did edit the description!

Comment (TheDarkElites) [reply code/game/objects/items/weapons/storage/boxes.dm]: Oh yeah you did idk why github bugged out and showed me the old one

**Pull request closed without merging**